src/ckeditorinspector.js

```diff
@@ -10,16 +10,12 @@ import ReactDOM from 'react-dom';
 
 import InspectorUI from './components/ui';
 import Logger from './logger';
+import { normalizeArguments } from './utils';
 import './ckeditorinspector.css';
 
 // From changelog -> webpack.
 window.CKEDITOR_INSPECTOR_VERSION = CKEDITOR_INSPECTOR_VERSION;
 
-const container = document.createElement( 'div' );
-container.className = 'ck-inspector-wrapper';
-
-let unnamedEditorCount = 0;
-
 export default class CKEditorInspector {
```

Comment:
Could the `CKEditorInspector` throw an error in the constructor - I guess it might be a common misusage.

Reply:
Like when people use it `new CKEditorInspector( editor )`?

Reply:
I used `console.warn()` instead.